src/algorithms/calorimetry/CalorimeterIslandCluster.cc

```diff
@@ -20,19 +20,66 @@
 
 using namespace edm4eic;
 
-//
-// This algorithm converted from:
-//
-//  https://eicweb.phy.anl.gov/EIC/juggler/-/blob/master/JugDigi/src/components/CalorimeterHitDigi.cpp
-//
-// TODO:
-// - Array type configuration parameters are not yet supported in JANA (needs to be added)
-// - Random number service needs to bew resolved (on global scale)
-// - It is possible standard running of this with Gaudi relied on a number of parameters
-//   being set in the config. If that is the case, they should be moved into the default
-//   values here. This needs to be confirmed.
+static double Phi_mpi_pi(double phi) {
+  return std::remainder(phi, 2 * M_PI);
+}
 
+//TODO:Reconcile edm4hep::Vector2f and edm4eic::Vector3f especially with regards to the operators and sign convention
+static edm4hep::Vector2f localDistXY(const CaloHit &h1, const CaloHit &h2) {
+  //edm4eic::Vector3f h1_pos=geo_converter->position(h1.getCellID());
+  //edm4eic::Vector3f h2_pos=geo_converter->position(h2.getCellID());
+  const auto delta =h1.getLocal() - h2.getLocal();
+  return {delta.x, delta.y};
+  //const auto deltax = h1.getLocal()[0] - h2.getLocal()[0];
+  //const auto deltay = h1.getLocal()[1] - h2.getLocal()[1];
+  //return {delta.x, delta.y,0};
+  //return {deltax,deltay};
```

Comment:
Did you want to keep this around for the `TODO` above?

Reply:
I'm not sure what those comments are about. We probably don't need to consider those TODOs.

Reply:
I'd remove these comments while we're doing cleanup.

Reply:
Done